internal/pkg/service/common/httpserver/middleware/otel_test.go

```diff
@@ -128,10 +137,13 @@ func expectedSpans(tel telemetry.ForTest) tracetest.SpanStubs {
 				attribute.String("http.query.foo", "bar"),
 				attribute.String("http.query.secret2", "****"),
 				attribute.String("http.header.x-storageapi-token", "****"),
-				attribute.String("resource.name", "/api/item/:id/:secret1"),
+				attribute.String("resource.name", "/api/item/:id/:secret1 MyEndpoint"),
 				attribute.String("http.route", "/api/item/:id/:secret1"),
 				attribute.String("http.route_param.id", "123"),
 				attribute.String("http.route_param.secret1", "****"),
+				attribute.String("endpoint.service", "MyService"),
+				attribute.String("endpoint.name", "MyEndpoint"),
+				attribute.String("endpoint.name_full", "MyService.MyEndpoint"),
 				attribute.String("http.response.header.x-request-id", "<dynamic>"),
 				attribute.Int("http.wrote_bytes", 10),
```

Comment:
A testuje sa to cele v `otel_test.go`, vsetky 3 otel middlewares, takto sa to viac podoba produkcnemu setupu.